Dockerfile

```
@@ -0,0 +1,18 @@
+FROM docker.io/golang:1.16 as build
+ARG VERSION
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o ktr
+
+FROM gcr.io/distroless/static:nonroot
+
+COPY --from=build /app/ktr /usr/local/bin/
+
+ENTRYPOINT [ "/usr/local/bin/ktr" ]
```

In GH release action, the caching of Go packages is useless if we are downloading the internet again in the docker build. Goreleaser uses an approach where it prebuilds the binary and then copies the file from "outside". An example is here: https://github.com/vshn/k8up/blob/master/Dockerfile
```suggestion
FROM gcr.io/distroless/static:nonroot

COPY kube-token-refresher /usr/local/bin/ktr

ENTRYPOINT [ "/usr/local/bin/ktr" ]
```